source/index.html.md.erb

```diff
@@ -16627,24 +17141,35 @@ IndexResult = ss_client.Webhooks.list_webhooks(
 	"pageSize": 100,
 	"totalPages": 1,
 	"totalCount": 1,
-	"data": [
-		{
-			"id": 4503604829677444,
-			"name": "Webhook #5",
-			"scope": "sheet",
-			"scopeObjectId": 6761855684241284,
-			"events": [
-				"*.*"
-			],
-			"callbackUrl": "http://www.myApp.com/webhooks",
-			"sharedSecret": "216ejjzdnq17mq1q8xs7d4hu8b",
-			"enabled": true,
-			"status": "ENABLED",
-			"version": 1,
-			"createdAt": "2015-11-03T14:52:21Z",
-			"modifiedAt": "2015-11-04T19:05:40Z"
-		}
-	]
+    "data": 
+    [{
+        "id": 4503604829677444,
+        "name": "Webhook #5",
+        "scope": "sheet",
+        "scopeObjectId": 6761855684241284,
+        "events": [ "*.*" ],
+        "callbackUrl": "http://www.myApp.com/webhooks",
+        "sharedSecret": "216ejjzfoo17mq1q8xs7d4hu8b",
+        "enabled": true,
+        "status": "ENABLED",
+        "version": 1,
+        "createdAt": "2015-11-03T14:52:21Z",
+        "modifiedAt": "2015-11-04T19:05:40Z"
+    },
+    {
```

Comment:
Not sure there's any benefit to showing two items. (And doesn't match 'totalCount')